CHANGELOG.rst

```diff
@@ -16,10 +16,20 @@ The format is based on `Keep a Changelog`_.
   - **Fixed**: for any bug fixes.
   - **Security**: in case of vulnerabilities.
 
-
 `UNRELEASED`_
 =============
 
+
+`0.7.2`_ - 2019-03-05
+=====================
+
+
+Fixed
+-----
+
+- Fixed a bug whereby inheriting from QuantumRegister or ClassicalRegister
+  would cause a QiskitError in instruction.py
```

Comment:
This should be moved to the existing unreleased Fixed section which is below and already has a few items in the list. Also you can add `(#1908)` or ` (#1892)` to the end of the line.

Reply:
Can do